go.sum

```diff
@@ -1,4 +1,6 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898 h1:SC+c6A1qTFstO9qmB86mPV2IpYme/2ZoEQ0hrP+wo+Q=
```

Comment:
Why this file is changed that much?

Reply:
The go.sum file changed automatically when you add or remove something from go.mod file. Also when you run `go mod tidy` to clean up the mess from modules, the go.sum file also could be changed.